frontend/package.json

```diff
@@ -34,4 +34,4 @@
     "style-loader": "^3.3.1",
     "tailwindcss": "^3.0.12"
   }
-}
+}
```

Comment:
unneeded change here

Reply:
I think it's the auto indentation cause the problem.